README.md

```diff
@@ -1,114 +1,112 @@
+First, go to the release page for **[QuestCraft](https://github.com/QuestCraftPlusPlus/QuestCraft/releases/latest)** and download the APK to your computer or Android device.
 
-Next, open your headset's QuestCraft launcher (through Unknown Sources). Be sure to grant all requested permissions. Once you're done, you'll follow the on-screen prompts to enter your Minecraft login details, then press play and wait for all of the files to download (aproxx. 10 minutes).  (You must do this for each specific version to play that version!)
+Secondly, you'll need to install the QuestCraft APK to your VR Headset using **[SideQuest](https://sidequestvr.com/setup-howto)** and follow the necessary steps to connect your Quest. Drag and drop the APK, and wait for it to install.
 
-**NOTE: You may encounter crashes when downloading your selected MC version, ESPECIALLY on v53 and later. This is because Meta can't fix memory leaks, for some reason. Don't fret; launch the app again, press play again, and the download will continue where it left of before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the QuestCraft Discord (discord.gg/questcraft)**
+Next, open your headset's QuestCraft launcher (through Unknown Sources). Be sure to grant all requested permissions. Once you're done, you'll follow the on-screen prompts to enter your Minecraft login details, then press play and wait for all of the files to download (approx. 5-10 minutes).  (You must do this for each specific version to play that version!)
 
 And boom! You have successfully installed QuestCraft on your VR headset! To play, press the **Play** button and wait for it to load; depending on the headset's age, it might take a while.
 
-## Supported Renderers (Take note when installing mods!)
+**NOTE:** You may encounter crashes when downloading your selected MC version, mainly on v53 and later. This happens because Meta can't fix memory leaks for some reason. Don't fret; launch the app, press play, and the download will continue from where it left off before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the QuestCraft Discord (discord.gg/questcraft)
 
-- ### Zink
-**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, works with many mods, at the cost of performance. (which is mostly fixed with the performance mods below)
-- ### Vulkan
-**[Vulkan](https://www.vulkan.org/)**, while very fast, it may only works with a small amount of mods. For a list, please see the pinned discussion in #discussion on the **[VulkanMod Server](https://discord.gg/FVXg7AYR2Q)**. VulkanMod is not affiliated with us, so please don't ask and spam the devs questions related to QuestCraft! 
-*Vulkan is not yet out for QuestCraft, please wait!*
+### Please do not enable seated mode in the in-game VR settings!
 
 ## Included Mods 
```

Comment:
why is the section called included mods if its mostly to assist with installing mods